frappe/desk/page/setup_wizard/install_fixtures.py

```diff
@@ -2,30 +2,171 @@
 # License: GNU General Public License v3. See license.txt
 
 from __future__ import unicode_literals
-
 import frappe
-
+import json
 from frappe import _
+import copy
+
 
 def install():
-	update_genders_and_salutations()
+    update_genders_and_salutations()
+    make_administrative_area_fixtures()
+
 
 @frappe.whitelist()
 def update_genders_and_salutations():
-	default_genders = [_("Male"), _("Female"), _("Other")]
-	default_salutations = [_("Mr"), _("Ms"), _('Mx'), _("Dr"), _("Mrs"), _("Madam"), _("Miss"), _("Master"), _("Prof")]
-	records = [{'doctype': 'Gender', 'gender': d} for d in default_genders]
-	records += [{'doctype': 'Salutation', 'salutation': d} for d in default_salutations]
-	for record in records:
-		doc = frappe.new_doc(record.get("doctype"))
-		doc.update(record)
-
-		try:
-			doc.insert(ignore_permissions=True)
-		except frappe.DuplicateEntryError as e:
-			# pass DuplicateEntryError and continue
-			if e.args and e.args[0]==doc.doctype and e.args[1]==doc.name:
-				# make sure DuplicateEntryError is for the exact same doc and not a related doc
-				pass
-			else:
-				raise
\ No newline at end of file
+    default_genders = [_("Male"), _("Female"), _("Other")]
+    default_salutations = [_("Mr"), _("Ms"), _('Mx'), _("Dr"), _("Mrs"), _("Madam"), _("Miss"), _("Master"), _("Prof")]
+    records = [{'doctype': 'Gender', 'gender': d} for d in default_genders]
+    records += [{'doctype': 'Salutation', 'salutation': d} for d in default_salutations]
+    for record in records:
+        doc = frappe.new_doc(record.get("doctype"))
+        doc.update(record)
+
+        try:
+            doc.insert(ignore_permissions=True)
+        except frappe.DuplicateEntryError as e:
+            # pass DuplicateEntryError and continue
+            if e.args and e.args[0] == doc.doctype and e.args[1] == doc.name:
+                # make sure DuplicateEntryError is for the exact same doc and not a related doc
+                pass
+            else:
+                raise
+
+
+def make_administrative_area_fixtures():
+
+    administrative_areas = get_administrative_areas()
```

Comment:
Only install if region is India! (Every region will have its own administrative areas)